packages/commerce-sdk-react/.eslintignore

```diff
@@ -0,0 +1,8 @@
+coverage
```

Comment:
A lot of new files added to this PR because I brought over `commerce-sdk-react`, which was not available in the branch for PWA Kit v2. 

The plan is to have them here in this PR for only testing purpose. I'll remove them later before merging the PR, and there will be another PR to have `commerce-sdk-react` package be merged into the `develop` branch.